### PR TITLE
Closes #73 prototype graduate page

### LINKED ISSUE #73: Prototype Graduates page
Using existing components (not YAML) for comparison with built version.

### PR DESCRIPTION
**Pull request by sjmog**
Closes #73

### CODE REVIEW

File: source/components/templates/holler/_call_to_action.erb [hunk 1]
```diff
@@ -22,7 +23,7 @@
     <section class="holler-block__section">
       <%= partial :"groups/cta-block", locals: {
         modifiers: "inline",
-        ctas: ctas.map{ | cta | cta.merge(modifiers: "horizontal") }
+        ctas: ctas.each { |cta| cta[:modifiers] ? cta : cta[:modifiers] = "horizontal" }
```
Comment (pitchinvasion) [inline source/components/templates/holler/_call_to_action.erb]: You shouldn't mutate something you're iterating over, I definitely prefer map here, you could always do `cat[:modifiers] ? cat: cta.merge(modifiers: "horizonal")`

Comment (sjmog) [inline source/components/templates/holler/_call_to_action.erb]: I hear you about mutated state but these two lines don't do the same things though :(

I needed something that would iterate through the `ctas` and either leave the `:modifiers` alone (if they exist), or default them to `horizontal` (if they don't exist).

The line I removed just replaces *any* provided `modifiers` with "horizontal", which limits how much we can style the CTAs in this block.

The line I added checks to see if `modifiers` is given first, and either leaves it alone (if it is given), or sets `modifiers` to `"horizontal"`. The alternative is (I guess) that we do this:

```erb
<% modifiers = locals.fetch(:modifiers, "horizontal") %>
```

in the top of the _cta.erb atom file.

Comment (pitchinvasion) [inline source/components/templates/holler/_call_to_action.erb]: Ah I much prefer adding it in the cta file. Occam's razor wins again

Comment (sjmog) [inline source/components/templates/holler/_call_to_action.erb]: OK, I did it (but now we have this variable assignment in a view). Not totally convinced that isn't less smelly that mutating all the state :(

**Comment by sjmog** [comment]
:+1:

**Pull request merged**